README.md

```diff
@@ -35,34 +38,44 @@ npm link ../path/to/where/you/cloned/dwn-sdk-js
 ### nodeJS
 
 - **ESM**
+
   ```javascript
-  import { Dwn } from 'dwn-sdk';
+  import { Dwn } from "dwn-sdk";
 
   // cool things
   ```
 
 - **CJS**
+
   ```javascript
-  const { Dwn } = require('dwn-sdk');
+  const { Dwn } = require("dwn-sdk");
 
   // cool things
   ```
+
 ### Browser:
 
 - **UMD Bundle**
+
   ```html
-  <script type="text/javascript" src="node_modules/dwn-sdk/dist/bundles/bundle.umd.js"></script>
+  <script
+    type="text/javascript"
+    src="node_modules/dwn-sdk/dist/bundles/bundle.umd.js"
+  ></script>
   ```
 
 - **ESM Bundle**
   ```html
-  <script type="text/javascript" src="node_modules/dwn-sdk/dist/bundles/bundle.esm.js"></script>
+  <script
+    type="text/javascript"
+    src="node_modules/dwn-sdk/dist/bundles/bundle.esm.js"
+  ></script>
```

Comment:
Why the change to double quotes and formatting? Consider undo.

Reply:
not to sure, i ran n lint:fix and this was the output